src/Geometry.cc

```diff
@@ -308,6 +309,35 @@ void Geometry::SetPolylineShape(const std::vector<Polyline> &_polylines)
   this->dataPtr->polylines = _polylines;
 }
 
+std::optional<gz::math::Inertiald> Geometry::CalculateInertial(
+  double _density)
+{
+  std::optional< gz::math::Inertiald > geomInertial;
```

Comment:
```suggestion
  std::optional<gz::math::Inertiald> geomInertial;
```

Reply:
Corrected in [8fef740](https://github.com/gazebosim/sdformat/pull/1299/commits/8fef7400ad028ca9015c8454e52e713a01f0129c)